app/controllers/help_controller.rb

```diff
@@ -20,6 +20,8 @@ def show
 
   def ab_testing
     @ab_variant = request.headers["HTTP_GOVUK_ABTEST_EXAMPLE"] == "B" ? "B" : "A"
+
+    response.headers['Vary'] = 'GOVUK-ABTest-Example'
```

Comment:
It seems a bit dangerous to _set_ the `Vary` header rather than add it to existing headers, but it currently doesn't overwrite anything, and it's only affecting the A/B test page. The `Vary:Accept-Encoding` header is still being added correctly by Rails.

Rails 5 has an `add` method which lets you set multi-valued headers more safely.